pkg/controller/observabilityendpoint/metrics_collector.go

```diff
@@ -112,8 +112,7 @@ func createDeployment(clusterID string, obsAddonSpec oav1beta1.ObservabilityAddo
 	}
 
 	commands := []string{
-		"/usr/bin/telemeter-client",
-		"--id=$(ID)",
```

Comment:
I wonder why we remove `--id=$(ID)`, thanks

Reply:
@songleo 
this is inherited from telemeter client and in fact we don't need this